### PR TITLE
Add Mus2016

### PR DESCRIPTION
**Pull request by hagenw**
I just created a pull request for easier discussion.

### CODE REVIEW

File: massdatasets.py [hunk 1]
```diff
@@ -113,3 +143,75 @@ def __init__(self,
...
+    '''
+    The naming conventions are strange I know.
+    Basically MUS2016 is correct, but the data files we have received and are
+    available for download use 2017.
+    '''
+
+    def __init__(self,
+                 base_path='/vol/vssp/maruss/data2/MUS2017',
+                 results_filename='sisec_mus_2017_full.csv'):
+        super(MUS2016, self).__init__(base_path)
```
Comment (hagenw) [inline massdatasets.py]: In [python 3](https://docs.python.org/3.0/library/functions.html#super) you can now also write:
```python
super().__init__(base_path)
```
which I find easier to read.

File: massdatasets.py [hunk 2]
```diff
@@ -41,22 +46,47 @@ def add_song(self, artist, title, style, audio_filepath, **kwargs):
...
+        frame.rename(columns={'index': 'source'}, inplace=True)
 
         frame['audio_filepath'] = ['/'.join((self.base_path, _))
                                    for _ in frame['audio_filepath']]
         frame['dataset'] = self.dataset
+
+        # This is clunky way to add features in long format
+        if features and include_features:
+            frame2 = pd.concat(features, copy=False)
+            cols1, cols2 = list(frame.columns), list(frame2.columns)
```
Comment (hagenw) [inline massdatasets.py]: `cols1` and `cols2` is only used in the if-statement below, so you could move this line inside that statement.

File: massdatasets.py [hunk 3]
```diff
@@ -41,22 +46,47 @@ def add_song(self, artist, title, style, audio_filepath, **kwargs):
...
 
-    def to_pandas_df(self):
+    def to_pandas_df(self, include_features=True):
         '''
         Compiles the yaml document to a pandas DataFrame.
         audio_filepath are complete (prefixed by base_path).
         '''
 
-        frame = pd.DataFrame(columns=self.songs[0].keys())
+        long_format = True
```
Comment (hagenw) [inline massdatasets.py]: What is the meaning of `long_format`? You use the term also in a comment in line 72.

Comment (deeuu) [reply massdatasets.py]: Yeah I should add this as a parameter and include documentation.
But anyway: https://sejdemyr.github.io/r-tutorials/basics/wide-and-long/

File: massdatasets.py (hunk 1 above)
Comment (deeuu) [reply massdatasets.py]: Yeah but what about the old guys? Though I haven't even tested with Python 2.x yet...

File: massdatasets.py (hunk 2 above)
Comment (deeuu) [reply massdatasets.py]: Hopefully improved readability somewhat now

File: massdatasets.py (hunk 1 above)
Comment (hagenw) [reply massdatasets.py]: OK, I abandoned Python 2 ;)

**Pull request merged**